### ISSUE TITLE
unkinkPolygon does not handle polygons with interior holes

### ISSUE THREAD

**Issue by chrisbrainerd** [Turfjs/turf#1719]
When using `turf.unkinkPolygon()` on a polygon with interior rings, it appears to ignore those interior rings and treat them as exterior rings of an additional polygon. So for instance, if I feed in a FeatureCollection called `holeyFeatureCollection` to `unkinkPolygon(holeyFeatureCollection) which looks like this:
![image](https://user-images.githubusercontent.com/18501631/60291927-067aef80-98ea-11e9-8414-eccb7743f532.png)

this is what the function spits out: 
![image](https://user-images.githubusercontent.com/18501631/60291978-1f83a080-98ea-11e9-9692-380c224fbe8e.png)

Also as additional info, if I feed in just the single Feature with the holey polygon, or just the Polygon, the result is the same FeatureCollection as output.

Input: https://gist.github.com/chrisbrainerd/05bfae9c2b6420c69692c90877a9e1c2
Output: https://gist.github.com/chrisbrainerd/08e5ad1e401023ffdc48bf7d2fbb11bc

**Comment by rowanwins** [comment]
Yeah the original library used ([simple-polygon](https://github.com/mclaeysb/simplepolygon)) does provide information on the winding numbers which could used to reconstruct holes however turf is not using that info and instead strips it out which is problematic.

I'm currently working on a rewrite of the same algorithm which is much faster and I intend on providing an option which allows users to specify if they want to nest the holes accordingly. It's surprisingly complicated but I'm getting close!

**Comment by Stephane-Ag** [comment]
@rowanwins Any updates on this? This is still a problem for me now.
